benchmark.rb

```diff
@@ -0,0 +1,140 @@
+require 'json'
+require 'lightstep'
+require 'net/http'
+require 'pp'
+require 'uri'
+
+$base_url = "http://localhost:8000"
+
+$test_tracer = LightStep::Tracer.new(
+  component_name: 'lightstep/ruby/example',
+  transport: LightStep::Transport::HTTPJSON.new(
+    host: 'localhost',
+    port: 8000,
+    encryption: LightStep::Transport::HTTPJSON::ENCRYPTION_NONE,
+    access_token: 'none'
+  )
+)
+$noop_tracer = LightStep::Tracer.new(
```

Comment:
Yes, it does build the spans and send them to the transport which then doesn't transport them. I think the old code was like this too but I'm not certain.

Sounds like a good thing to do with the OT Ruby extract.